src/node/channels.h

```diff
@@ -514,6 +837,14 @@ namespace ccf
       }
     }
 
+    void restart()
```

Comment:
I cannot find where this is called?

Reply:
Only the channels test so far, I'm adding the logic for triggering it on the message counter now.